pixell/enmap.py

```diff
@@ -2729,7 +2729,9 @@ def fix_endian(map):
 	"""Make endianness of array map match the current machine.
 	Returns the result."""
 	if map.dtype.byteorder not in ['=','<' if sys.byteorder == 'little' else '>']:
-		map = map.byteswap(True).newbyteorder()
+		# Compatibility to numpy 2.0
```

Comment:
Could you explain this change a bit more? Might need @amaurea 's attention

Reply:
newbyteorder was removed in numpy 2.0. This is equivalent to the old code.

Reply:
There is now another branch (numpy-2p0-changes) that fixes the remainder of the warnings.